### PR TITLE
Archiving the project

### PR DESCRIPTION
**Pull request by SylLutze**
### Description
Archiving including adding the artifact, author info, updates 
- *Category*: archiving 

### Changes and notes
Adding the artifact files using the LFS 
Updates to the Readme and model spec file paths to reference the new artifact 
Adding the author info 
Updating the version of vivarium public health to solve a bug fix

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -103,22 +103,19 @@ You'll find six directories inside the main
...
 
-  (vivarium_nih_us_cvd) :~$ make_specs -v
-  2020-06-18 18:18:28.311 | 0:00:00.679701 | build_model_specifications:48 - Writing model spec(s) to "/REPO_INSTALLATION_DIRECTORY/vivarium_nih_us_cvd/src/vivarium_nih_us_cvd/model_specifications"
+Once this is updated, you can run your simulation from the command line. 
+With your conda environment active, you can run with, e.g.::
 
-As the log message indicates, the model specifications will be written to
-the ``model_specifications`` subdirectory in this repository. You can then
-run simulations by, e.g.::
+   (vivarium_nih_us_cvd) :~$ simulate run -vvv /<REPO_INSTALLATION_DIRECTORY>/vivarium_nih_us_cvd/src/vivarium_nih_us_cvd/model_specifications/nih_us_cvd.yaml -o /FILE/PATH/TO/SAVE/RESULTS
```
Comment (zmbc) [inline README.rst]: If I recall correctly, it is possible to specify the artifact on the command line, which then wouldn't require them to edit the YAML file? Look at `simulate run --help` to see if I am making this up!

**Review by zmbc — APPROVED** [review]
:tada:

File: README.rst (hunk 1 above)
Comment (SylLutze) [reply README.rst]: You are correct! Thanks, I'll edit this to have the artifact be part of the simulate call instead

File: .zenodo.json [hunk 2]
```diff
@@ -3,12 +3,42 @@
...
+        {
+            "orcid": "0009-0009-3789-5382",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Caroline Kinuthia"
+        },
+        {
+            "orcid": "0000-0001-6033-4713",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Abraham Flaxman"
+        }
```
Comment (rmudambi) [inline .zenodo.json:50]: Matt Kappel and Kjell Swedin also contributed to this model, so they should probably be added here if we have their orcids

File: setup.py [hunk 3]
```diff
@@ -45,7 +45,7 @@
     install_requirements = [
         "gbd_mapping>=3.1.0, <4.0.0",
         "vivarium>=2.1.1",
-        "vivarium_public_health>=2.0.0",
+        "vivarium_public_health>=2.2.0",
```
Comment (rmudambi) [inline setup.py]: For archival this should be pinned with ==, i.e. 
```suggestion
        "vivarium_public_health==2.2.0",
```

**Review by rmudambi — APPROVED** [review]

**Pull request merged**